main.nf

```diff
@@ -0,0 +1,932 @@
+-*- mode: groovy;-*-
+
+========================================================================================
+               NGI - E X O S E Q    B E S T    P R A C T I C E
+========================================================================================
+ #### Homepage / Documentation
+ https://github.com/scilifelab/NGI-ExoSeq
+ #### Authors
+ Senthilkumar Panneerselvam @senthil10 <senthilkumar.panneerselvam@scilifelab.se>
+ Phil Ewels @ewels <phil.ewels@scilifelab.se>
+ Alex Peltzer @alex_peltzer <alexander.peltzer@qbic.uni-tuebingen.de>
+ Marie Gauder <marie.gauder@student.uni-tuebingen.de>
+----------------------------------------------------------------------------------------
+Developed based on GATK's best practise, takes set of FASTQ files and performs:
+ - alignment (BWA)
+ - recalibration (GATK)
+ - realignment (GATK)
+ - variant calling (GATK)
+ - variant evaluation (SnpEff)
+*/
+
+// Package version
+version = '0.8'
+
+// Help message
+helpMessage = """
+===============================================================================
+NGI-ExoSeq : Exome/Targeted sequence capture best practice analysis v${version}
+===============================================================================
+
+Usage: nextflow scilifelab/NGI-ExoSeq --reads '*_R{1,2}.fastq.gz' --genome GRCh37
+
+This is a typical usage where the required parameters (with no defaults) were
+given. The available paramaters are listed below based on category
+
+Required parameters:
+--reads                        Absolute path to project directory
+--genome                       Name of iGenomes reference
+
+Output:
+--outdir                       Path where the results to be saved [Default: './results']
+
+Kit files:
+--kit                          Kit used to prep samples [Default: 'agilent_v5']
+--bait                         Absolute path to bait file
+--target                       Absolute path to target file
+--target_bed                   Absolute path to target bed file (snpEff compatible format)
+
+Genome/Variation files:
+--dbsnp                        Absolute path to dbsnp file
+--thousandg                    Absolute path to 1000G file
+--mills                        Absolute path to Mills file
+--omni                         Absolute path to Omni file
+--gfasta                       Absolute path to genome fasta file
+--bwa_index                    Absolute path to bwa genome index
+
+Other options:
+--project                      Uppnex project to user for SLURM executor
+
+For more detailed information regarding the parameters and usage refer to package
+documentation at https:// github.com/scilifelab/NGI-ExoSeq
+"""
+
+// Variables and defaults
+params.help = false
+params.reads = false
+params.singleEnd = false
+params.genome = false
+params.run_id = false
+params.exome = true //also default, integrate whole genome later
+params.aligner = 'bwa' //Default, but stay tuned for later ;-) 
+params.saveReference = true
+
+//Clipping options
```

Comment:
Personally, I prefer having spaces after comment `//`s 🤓

Reply:
760b23db318593257c8f5a46d7b1bfa25dc65fee fixes that ;-)